### PR TITLE
Fix channel switching and handle no channels.

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

Fixed channel switching bugs introduced by merge.

## Reviewer guidance

I hid the entire toolbar toolbar since breadcrumbs and the search button would not be useful if no channels existed.

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/actions.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ function _getCurrentChannelId() {
   return new Promise((resolve, reject) => {
     _getChannelList()
       .then((channelList) => {
-        if (channelList.length) {
+        if (channelList) {
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/actions.js:96]: you may want to check both `if (channelList && channelList.length)`?

```
[] ? 'yes' : 'no'
"yes"
[].length ? 'yes' : 'no'
"no"
```

File: kolibri/plugins/learn/assets/src/vue/toolbar/index.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
+  <div v-if="channelsExist" v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/vue/toolbar/index.vue]: same here

File: kolibri/plugins/learn/assets/src/actions.js (hunk 1 above)
Comment (christianmemije) [inline kolibri/plugins/learn/assets/src/actions.js:96]: Problem was that channelList was null so it failed at checking the length of a null array.

Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/actions.js:96]: right, but now if you get passed an empty array somehow, the logic won't work. So you'll need to check both `channelList` (not null) and `channelList.length` (not empty)

**Pull request merged**